metafacture-biblio/src/test/java/org/metafacture/biblio/marc21/MarcXmlHandlerTest.java

```diff
@@ -146,6 +146,63 @@ public void issue330ShouldOptionallyRecognizeRecordsWithoutNamespace()
         verifyNoMoreInteractions(receiver);
     }
 
+    @Test
+    public void shouldRecognizeRecordsWithoutNamespace()
+        throws SAXException {
```

Comment:
Wrong indentation for all `throws` qualifiers.

Reply:
Fixed in [8305caf](https://github.com/metafacture/metafacture-core/pull/575/commits/8305cafd27ab202bff8877a96f478790b83ceed8).